### PR TITLE
[RA1 Ch02] RST conversion: corrections

### PR DESCRIPTION
**Pull request by karinesevilla**
Addition of backslashes to fix cut words

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.rst [hunk 1]
```diff
@@ -3666,18 +3660,18 @@ Compliance with Standards (source `RM 7.9.10 <../../../ref_model/chapters/chapte
...
+| se\       | Standards         | The Cloud Operator, |               |
 | c.std.006 |                   | Platform and        |               |
 |           |                   | Workloads           |               |
 |           |                   | **should** utilise  |               |
 |           |                   | the `OWASP Web      |               |
 |           |                   | Security Testing    |               |
-|           |                   | Guid                |               |
-|           |                   | e <https://github.c |               |
+|           |                   | Guide               |               |
+|           |                   | <https://github.c   |               |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.rst]:
```suggestion
|           |                   | <https://github.c\   |               |
```

**Comment by pgoyal01** [comment]
@karinesevilla Great Job -- Thanks. Just one minor typo,

**Review by pgoyal01 — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

File: doc/ref_arch/openstack/chapters/chapter02.rst (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter02.rst]: @pgoyal01 suggestion for the word "Guide" integrated.
No backslash needed into URL.

Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.rst]: @karinesevilla There is now a space after ".c" and before "om/OWASP..."

**Review by collivier — APPROVED** [review]

**Pull request merged**